tests/components/charts.test.ts

```diff
@@ -141,6 +141,38 @@ describe("figures", () => {
     }
   );
 
+  test.each(["basicChart", "scorecard", "gauge"])(
+    "chart menu position is correct when clicking on menu button",
+    async (chartType: string) => {
+      mockGetBoundingClientRect({
+        "o-spreadsheet": () => ({ top: 25, left: 25, height: 1000, width: 1000 }),
+        "o-figure-menu-item": () => ({ top: 500, left: 500 }),
+      });
+      createTestChart(chartType);
+      await nextTick();
+      await simulateClick(".o-figure-menu-item");
+      const menuPopover = fixture.querySelector<HTMLElement>(".o-popover")!;
+      expect(menuPopover.style.top).toBe(`${500 - 25}px`); // 25 : spreadsheet offset
```

Comment:
Maybe a constant like SPREADSHEET_OFFSET should be nice if we have to change it one day ?

Reply:
The offset is defined in the `mockGetBoundingClientRect` 5 lines above 🙃 A constant seems overkill, I'll update teh comment to be clearer